detection-rules/nlu_preview_attachment_callback_scam.yml

```diff
@@ -0,0 +1,31 @@
+name: "Attachment: Callback scam from first-time sender"
+description: |
+    Detects callback scams by analyzing text within images of receipts or invoices from first time senders.
+type: "rule"
+severity: "low"
+source: |
+  type.inbound
+  and any(attachments, .file_type in ("png", "jpg")
```

Comment:
@bfilar will this rule work with PDFs also? i.e. `pdf`

trying to replicate this rule here: https://github.com/sublime-security/sublime-rules/blob/main/detection-rules/attachment_invoice_fake_customer_service_freemail_sender.yml#L11